packages/components/pie-button/src/button.scss

```
@@ -108,16 +109,27 @@ $breakpoint-wide: 768px;
 
             @include p.button-interactive-states('--dt-color-interactive-primary', 'inverse');
         }
+
+        // This is to handle the background colour change getting overridden above, but then being put in responsive mode
+        // can't use :not for this unfortunately, as then the smaller narrow state would lose its background colour too
+        &.o-btn--xsmall.is-responsive.o-btn--expressive,
+        &.o-btn--small-productive.is-responsive {
```

Does `.is-responsive` already come from the mixin?
```suggestion
        &.o-btn--xsmall.o-btn--expressive,
        &.o-btn--small-productive {
```

---

Just checked, and yep it does 😆  Good spot

---

Fixed